pegasus/helpers/form_helpers.rb

```diff
@@ -79,6 +79,10 @@ def insert_or_upsert_form(kind, data, options={})
   row
 end
 
+# @param [String] kind The kind of the form to be updated.
+# @param [String] secret The secret associated with the form to be updated.
+# @param [Hash] data The data to update.
+# @return [Hash] The hash of values sent to the DB for updating.
```

Comment:
Yeah, I did it wrong for some time too. And I, too, find it less readable this way.